src/System.IO.Ports/tests/SerialPort/Write_byte_int_int_Generic.cs

```diff
@@ -6,13 +6,14 @@
 using System.IO.PortsTests;
 using System.Text;
 using System.Threading;
+using System.Threading.Tasks;
 using Legacy.Support;
 using Xunit;
 using Xunit.NetCore.Extensions;
-using ThreadState = System.Threading.ThreadState;
 
 namespace System.IO.Ports.Tests
 {
+    [Trait("Crash", "True")]
```

Comment:
@danmosemsft Thanks for spotting those somehow I missed those in my grep pass. Already fixed on latest.